diff --git a/dist/main.cjs.js b/dist/main.cjs.js
index 7c8cdc59fc04d05308d2799d283b01c934d6c46f..d68acf75b3382d38c6286a00ec71d9bd0e98a8af 100644
--- a/dist/main.cjs.js
+++ b/dist/main.cjs.js
@@ -6428,6 +6428,10 @@ function parse$6(value, settings) {
 
         namedEntity = type === name$1 ? decodeEntity(characters) : false;
 
+        // QUICKFIX https://github.com/pngwn/MDsveX/issues/296
+        if (namedEntity == '<') namedEntity = '&lt;';
+        if (namedEntity == '>') namedEntity = '&gt;';
+
         if (namedEntity) {
           entityCharacters = characters;
           entity = namedEntity;
