diff --git a/node_modules/mdsvex/dist/main.cjs.js b/node_modules/mdsvex/dist/main.cjs.js
index 9755b4d..a87d82a 100644
--- a/node_modules/mdsvex/dist/main.cjs.js
+++ b/node_modules/mdsvex/dist/main.cjs.js
@@ -8721,6 +8721,10 @@ function parse$7(value, settings) {
 
         namedEntity = type === name$1 ? decodeEntity_1(characters) : false;
 
+        // QUICKFIX https://github.com/pngwn/MDsveX/issues/296
+        if (namedEntity == '<') namedEntity = '&lt;';
+        if (namedEntity == '>') namedEntity = '&gt;';
+
         if (namedEntity) {
           entityCharacters = characters;
           entity = namedEntity;
